src/Enum.php

```diff
@@ -80,15 +80,18 @@ public function __toString()
     }
 
     /**
-     * Compares one Enum with another.
+     * Determines if Enum should be considered equal with the variable passed as a parameter.
+     * Returns false if an argument is an object of different class or not an object.
      *
      * This method is final, for more information read https://github.com/myclabs/php-enum/issues/4
      *
-     * @return bool True if Enums are equal, false if not equal
+     * @return bool
      */
-    final public function equals(Enum $enum = null)
+    final public function equals($variable = null)
```

Comment:
Why is the `= null` default kept? `$enum->equals()` seems like a pretty strange call to make.

Reply:
This would be a BC break.